### PR TITLE
Restaurant menu page refactoring

### PR DESCRIPTION
**Pull request by andriifromlviv**

### CODE REVIEW

File: TAF_EasyRest/TestFramework/Pages/Client/RestaurantMenuPage.cs [hunk 1]
```diff
@@ -30,5 +21,14 @@ public CartPageComponent ClickShowCartButton()
             _showCartButton.Click();
             return new CartPageComponent(driver);
         }
+
+        public RestaurantMenuPage AddTheFistDishToTheCart()
+        {
+            var _menuItem = _menuItemSummaryCards.ElementAt(0);
```
Comment (dmatsev) [inline TAF_EasyRest/TestFramework/Pages/Client/RestaurantMenuPage.cs:27]: local variables should start with low letter without underscore, please rename the variables

File: TAF_EasyRest/TestFramework/Pages/Client/RestaurantMenuPage.cs [hunk 2]
```diff
@@ -30,5 +21,14 @@ public CartPageComponent ClickShowCartButton()
...
+            
+            var _addToCartButton = _menuItem.FindElement(By.XPath(".//button[contains(@class,'addButton')]"));
```
Comment (dmatsev) [inline TAF_EasyRest/TestFramework/Pages/Client/RestaurantMenuPage.cs:29]: I believe you've forgot to click on the button :)

**Review by dmatsev — CHANGES_REQUESTED** [review]
Please resolve the comments.

**Review by LoveMyDaughter — DISMISSED** [review]
Please resolve Dmytro's comments.

File: TAF_EasyRest/TestFramework/Pages/Client/RestaurantMenuPage.cs (hunk 1 above)
Comment (andriifromlviv) [reply TAF_EasyRest/TestFramework/Pages/Client/RestaurantMenuPage.cs:27]: done

File: TAF_EasyRest/TestFramework/Pages/Client/RestaurantMenuPage.cs (hunk 2 above)
Comment (andriifromlviv) [reply TAF_EasyRest/TestFramework/Pages/Client/RestaurantMenuPage.cs:29]: done)

**Review by dmatsev — APPROVED** [review]

**Review by LoveMyDaughter — APPROVED** [review]

**Pull request merged**